### PR TITLE
Update roadmap sprint 2 Q1

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Sprint 2 Updates

### CODE REVIEW

File: deployed/doc-it/R19Roadmap.json [hunk 1]
```diff
@@ -112,15 +112,15 @@
         "description": "Designer templates / examples for Loading States",
         "year": 2021,
         "quarter": "Q1",
-        "status": "backlog",
+        "status": "pre-release",
```
Comment (joebochill) [inline deployed/doc-it/R19Roadmap.json:115]: still backlog

File: deployed/doc-it/R19Roadmap.json [hunk 2]
```diff
@@ -383,7 +383,7 @@
         "description": "Add a component for dropdown menu from avatar",
         "year": 2021,
         "quarter": "Q1",
-        "status": "backlog",
+        "status": "in-progress",
```
Comment (joebochill) [inline deployed/doc-it/R19Roadmap.json:386]: you can call this pre-release...i'm pretty sure it'll go out in beta today

**Review by jeffvg — APPROVED** [review]

**Review by joebochill — COMMENTED** [review]
- Angular CLI templates are finished too

File: deployed/doc-it/R19Roadmap.json [hunk 3]
```diff
@@ -96,7 +96,7 @@
         "description": "Designer templates / examples for searching and filtering",
```
Comment (huayunh) [inline deployed/doc-it/R19Roadmap.json:96]: We don’t really do “filtering” (besides like some simple drop downs) can we change the name of this item 🙈

**Review by joebochill — CHANGES_REQUESTED** [review]
See comments.

File: deployed/doc-it/R19Roadmap.json (hunk 2 above)
Comment (emclaug2) [reply deployed/doc-it/R19Roadmap.json]: This is now pre-release.

**Comment by emclaug2** [comment]
Comment changes should be addressed now.

**Review by joebochill — COMMENTED** [review]
You need to update the search pattern description for the 2021 entry. Also Angular CLI templates are 'finished'.

**Review by joebochill — APPROVED** [review]

**Comment by emclaug2** [comment]
> You need to update the search pattern description for the 2021 entry. Also Angular CLI templates are 'finished'.

Good catch, this is updated now.  I can see why splitting out the roadmap into separate files would be a good refactor now.

**Pull request merged**